src/components/notifications/SIGNL4.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+    <div class="mb-3">
+        <label for="webhook-url" class="form-label">{{ $t("SIGNL4 Webhook URL") }}</label>
+        <input
+            id="webhook-url"
+            v-model="$parent.notification.webhookURL"
+            type="url"
+            pattern="https?://.+"
+            class="form-control"
+            required
+        />
+    </div>
+
+    <div class="mb-3">
+        <label for="webhook-request-body" class="form-label">{{
+            $t("Alert Body")
+        }}</label>
+        <select
+            id="webhook-request-body"
```

Comment:
please replace the mentions of webhook with `signl4`

Reply:
Done. I hope I understood correctly.